packages/components/psammead-radio-schedule/src/index.jsx

```diff
@@ -1,5 +1,138 @@
 import React from 'react';
+import styled from 'styled-components';
+import { GEL_SPACING, GEL_SPACING_DBL } from '@bbc/gel-foundations/spacings';
+import Grid from '@bbc/psammead-grid';
+import { arrayOf, number, oneOf, shape, string } from 'prop-types';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import ProgramCard from './ProgramCard';
+import StartTime from './StartTime';
 
-const RadioSchedule = () => <div>Radio Schedule Package</div>;
+const StartTimeWrapper = styled.div`
+  padding: ${GEL_SPACING_DBL} 0 ${GEL_SPACING};
+`;
+
+const StyledGrid = styled(Grid)`
+  display: flex;
+  flex-direction: column;
+`;
+
+const renderSchedule = ({ service, script, dir, locale, program }) => {
+  const {
+    state,
+    stateLabel,
+    startTime,
+    link,
+    brandTitle,
+    episodeTitle,
+    summary,
+    duration,
+    durationLabel,
+  } = program;
+
+  // we should format the start-time here and pass the formatted start-time to the  program card
+  const formattedStartTime = '13:00';
```

Comment:
This formating will be handled in a separate issue https://github.com/bbc/psammead/issues/3036